model-optimizer/mo/back/preprocessing.py

```diff
@@ -0,0 +1,381 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import logging as log
+
+from mo.utils.error import Error
+from mo.utils.utils import refer_to_faq_msg
+
+import numpy as np
+
+from openvino.preprocess import PrePostProcessor        # pylint: disable=no-name-in-module,import-error
+# pylint: disable=no-name-in-module,import-error
+from openvino.runtime import Function, Layout, PartialShape, layout_helpers
+
+
+def update_mean_scale_to_dict(input_nodes: list, mean_scale_val, scale):
```

Comment:
please use `get_mean_scale_dictionary(mean_values, scale_values, argv_input: str)` from cli_parser.py. It looks duplicate